### ISSUE TITLE
Does kinto have no post-xkeysnail start/bootup concept?

### ISSUE THREAD

**Issue by joshgoebel** [rbreaves/kinto#721]
Right now I find myself needing to do several things after the keymapper starts (to re-configure the virtual devices):

- run `xmodmap` to teach X11 about my Hyper key
- run `xset r rate 200 20` to get proper (fast) key repeating on the virtual keyboard
- run `xinput set-prop 'pointer:Keyszer VIRTUAL Keyboard'` several times to configure the pointing device (that I proxy to avoid false Alt/Super keystrokes)

I'm guessing these aren't all needs most Kinto users have but surely someone has noticed the key repeat speed issue - every time the keymapper restarts it breaks the repeat speed (that I have configured) and that needs to be set again on the virtual keyboard.

Has this come up at all?  I'm trying to figure out the "best" way to deal with this - to hopefully make it seamless... what would be nice is that if the keymapper silently crashed (and restarted) once in a while that no one would notice... but right now it's impossible not to notice.  My whole WM broke last time because I lost my Hyper key during a restart... 😭 until I could run xmodmap to fix it...

**Comment by rbreaves** [comment]
Post? Not really lol. It’s been a perfect ride for me the moment it starts.

I have some personal pre-setup routine that I do on Windows keyboards that involves remapping via keytables binary on GitHub that’s a low level remap to get it to align w/ macOS keyboards.. besides that though.. there’s not really anything special that I do. It’s not part of Kinto but it could be optional.

I also avoid things that involve xmodmap/ xkb as I just try & do everything needed in xkeysnail. I’ll be honest though I don’t really understand the need or desire for a hyper key.. Ctrl, Alt & Super has always done it for me so a 4th modifier is a mystery to me.

I also have no clue about the need to proxy your pointing device :/. And until I did some VNC stuff recently I never had key repeat issues either. I fixed my issue though by increasing the “hold key delay” in my DE/OS settings. But that’s a VNC latency issue - nothing w/ xkeysnail.
